stdlib/LinearAlgebra/src/LinearAlgebra.jl

```diff
@@ -433,21 +433,56 @@ function versioninfo(io::IO=stdout)
     println(io, "LAPACK: ",Base.liblapack_name)
 end
 
-function __init__()
-    try
-        BLAS.check()
-        if BLAS.vendor() === :mkl
-            ccall((:MKL_Set_Interface_Layer, Base.libblas_name), Cvoid, (Cint,), USE_BLAS64 ? 1 : 0)
+function get_blas_lapack_path()
```

Comment:
@nalimilan This is the code that constructs the full paths to BLAS and LAPACK. Currently it only looks at the absolute paths LIBBLAS and LIBLAPACK inside Julia's build. So what we need is a way to communicate the full path at build time.

Reply:
If a path that is not absolute is passed in, that's fine, we just try to `dlopen()` it.  So as long as `dlopen()` can find it it should work.  I think we can just save whatever `LIBBLASNAME` is, and if it's absolute then great, if not, we just rely on `dlopen()` to find it.

Reply:
OK. Actually what I do know is that I create a symlink in `$(libdir)/julia` pointing to the library in `/usr/lib`. IIRC packages that use BLAS and are built using BinaryBuilder like Arpack.jl need that link to anyway. Though it would be nice if Julia could create that link automatically as it happens for most other libraries.

Reply:
The idea behind libblastrampoline is that all other packages (including things like `Arpack.jl`) will link against `libblastrampoline` instead, so that we can configure the backing BLAS library at runtime.

Reply:
Right, we should not need the symlink anymore.

Reply:
The symlink should continue to work as is, and I suppose the full path should also work.